scripts/bootstrap

```diff
@@ -7,11 +7,11 @@ SCRIPTS_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
 cd "$SCRIPTS_DIR/.."
 
 function run() {
-    1>&2 echo "running $@"
+    1>&2 echo "running $*"
```

Comment:
@rvanasa This was not working on my MAC when I was trying to reproduce the build and the linting in Intellij was notifying me that I should change this. Now this seems to work both on MAC and linux machines.